opentelemetry-user-events-trace/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "opentelemetry-user-events-trace"
```

Comment:
What do you think about using a single crate for `user_events` exporter with dedicated features for the different signals? I think that's a better user experience and it's also aligned with the SDK and other exporter crates (such as OTLP exporter).

Reply:
we should be able to do that. (even potentially merge etw/user-events). Not an issue to start separate, given each signals are different today in upstream.